dokka-subprojects/analysis-java-psi/src/main/kotlin/org/jetbrains/dokka/analysis/java/util/PsiCommentsUtils.kt

```diff
@@ -44,7 +43,10 @@ internal fun PsiElement.referenceElementOrSelf(): PsiElement? =
     } else this
 
 internal fun PsiDocTag.linkElement(): PsiElement? =
-    valueElement ?: dataElements.firstOrNull { it !is PsiWhiteSpace }
+    dataElements.firstOrNull {
+        it.node.elementType == JavaDocElementType.DOC_REFERENCE_HOLDER ||
+                it.node.elementType == JavaDocElementType.DOC_METHOD_OR_FIELD_REF
```

Comment:
Could you please add a comment on why those two specific types should be used here?
It's not really clear for me why only those two out of all `JavaDocElementType`s we should care about